### PR TITLE
[cmsp-403] Add success info

### PR DESCRIPTION
**Pull request by jazzsequence**
This PR adds a success message if the workflow was successfully triggered.
If there was a problem triggering the workflow, we may not get much info back, so we also display an error message with instructions to check the dashboard and try again.

https://www.loom.com/share/446b24c34fbd42d6ae084f0e88f9e6e8

### CODE REVIEW

**Review by pwtyler — APPROVED** [review]

**Comment by jazzsequence** [comment]
@pwtyler if you don't mind giving this one more check since it went through a lot ofiterations as I was debugging the failing tests

File: src/Commands/ScaffoldExtensionCommand.php [hunk 1]
```diff
@@ -62,17 +62,25 @@ public function runScaffoldExtensionsJob(string $site_info = '', $job_name = '')
         $site = $this->getSite($site_id);
         $env = $site->getEnvironments()->get($site_env);
 
+        // Bail if getSiteById doesn't exist.
+        if (! method_exists($this, 'getSiteByID')) {
+            $this->log()->error('This command requires Terminus 3.2.0 or later. You appear to be running an earlier version of Terminus. Please update Terminus with terminus self:update and try again.');
```
Comment (pwtyler) [inline src/Commands/ScaffoldExtensionCommand.php:67]: is terminus self:update the only correct way to use Terminus now? what about homebrew?
```suggestion
            $this->log()->error('This command requires Terminus 3.2.0 or later. You appear to be running an earlier version of Terminus. Please update Terminus and try again.');
```

File: src/Commands/ScaffoldExtensionCommand.php [hunk 2]
```diff
@@ -93,9 +101,18 @@ public function runScaffoldExtensionsJob(string $site_info = '', $job_name = '')
...
 
-        return $env->getWorkflows()->create('scaffold_extensions', compact('params'));
+        // Run the workflow and trigger a success message if it triggered successfully.
+        if ($env->getWorkflows()->create('scaffold_extensions', compact('params'))) {
+            $success_message = sprintf("The %s workflow has been started on %s.%s.\n You can see the workflow running on your dashboard: %s", $job_id, $site_id, $site_env, $dashboard_url);
+            $this->log()->notice($success_message);
+            return;
+        }
+
+        // We should never get to this path, but if we do, that means the workflow failed to trigger. We'll link the user to the dashboard and instruct them to just try it again since we don't really know what happened.
```
Comment (pwtyler) [inline src/Commands/ScaffoldExtensionCommand.php:113]: Because we never have sad ygg rest endpoints 😆

**Review by pwtyler — APPROVED** [review]

File: src/Commands/ScaffoldExtensionCommand.php (hunk 1 above)
Comment (jazzsequence) [reply src/Commands/ScaffoldExtensionCommand.php:67]: Good point, I can just say "update terminus" and be more general.

**Pull request merged**